src/Request/Request.php

```diff
@@ -358,6 +360,18 @@ public function getParsedBody()
     }
 
 
+    /**
+     * @return mixed[]
+     */
+    public function getParsedBodyAsArray(): array
+    {
+        $parsedBody = $this->getParsedBody();
+        assert(is_array($parsedBody));
```

Comment:
Is it always truth?

Reply:
nope, you should use it only in case you know it will be array...I want to use this method after validation is done, but as getParsedBody is mixed, I'm forced to assert array every time I want to work with parsed body even if I already now it is valid and pars-able to array :)

Reply:
So I don't like the name, it looks like it tries to parse body as array... Maybe `getArrayBody`?